app/tests/test_models.py

```diff
@@ -0,0 +1,241 @@
+        self.assertEqual(forget_password.otp, 123456)
+
+    def test_forget_password_str(self):
+        self.assertEqual(str(self.forget_password), str(self.forget_password.id))
+
+class CommunityModelTest(TestCase):
+
+    def setUp(self):
+        # Create a user
+        self.user = User.objects.create_user(username='testuser', email='testuser@example.com', password='password')
+
+        # Create a community
+        self.community = Community.objects.create(
+            Community_name='Test Community',
+            subtitle='Test Subtitle',
+            description='Test Description',
+            location='Test Location',
+            github='https://github.com/test',
+            email='test@example.com',
+            website='https://www.test.com',
+            user=self.user
+        )
+
+    def test_community_creation(self):
+        community = Community.objects.get(id=self.community.id)
+        self.assertEqual(community.Community_name, 'Test Community')
+        self.assertEqual(community.subtitle, 'Test Subtitle')
+        self.assertEqual(community.description, 'Test Description')
+        self.assertEqual(community.location, 'Test Location')
+        self.assertEqual(community.github, 'https://github.com/test')
+        self.assertEqual(community.email, 'test@example.com')
+        self.assertEqual(community.website, 'https://www.test.com')
+        self.assertEqual(community.user, self.user)
+
+    def test_community_str(self):
+        self.assertEqual(str(self.community), 'Test Community')
+
+class CommunityMemberModelTest(TestCase):
+
+    def setUp(self):
+        # Create a user
+        self.user = User.objects.create_user(username='testuser', email='testuser@example.com', password='password')
+
+        # Create another user
+        self.another_user = User.objects.create_user(username='anotheruser', email='anotheruser@example.com', password='password')
+
+        # Create a community
+        self.community = Community.objects.create(
+            Community_name='Test Community',
+            user=self.user
+        )
+
+        # Create a community member
+        self.community_member = CommunityMember.objects.create(
+            community=self.community,
+            user=self.user,
+            is_reviewer=False,
+            is_moderator=False,
+            is_admin=True
+        )
+
+    def test_community_member_creation(self):
+        community_member = CommunityMember.objects.get(id=self.community_member.id)
+        self.assertEqual(community_member.community, self.community)
+        self.assertEqual(community_member.user, self.user)
+        self.assertFalse(community_member.is_reviewer)
+        self.assertFalse(community_member.is_moderator)
+        self.assertTrue(community_member.is_admin)
+
+    def test_community_member_str(self):
+        self.assertEqual(str(self.community_member), f'{self.user} - {self.community}')
+
+    def test_unique_admin_per_community(self):
```

Comment:
Each community can have more than one admin. Please remove the intented exception error. The testcase works then